Puppetfile

```diff
@@ -3,13 +3,10 @@ forge 'https://forgeapi.puppetlabs.com'
 mod 'katello/common',                '>= 1.0.1 < 1.1.0'
 mod 'katello/service_wait',          '>= 1.0.3 < 1.1.0'
 
-mod 'katello/certs',                 '>= 3.0.0 < 3.1.0'
-mod 'katello/qpid',                  '>= 3.1.0 < 3.2.0'
-mod 'katello/pulp',                  '>= 5.0.0 < 5.1.0'
-mod 'katello/candlepin',             '>= 4.0.0 < 4.1.0'
+mod 'katello/certs',                 '>= 3.1.0 < 3.2.0'
+mod 'katello/qpid',                  '>= 4.0.0 < 4.1.0'
+mod 'katello/pulp',                  '>= 5.3.0 < 5.4.0'
+mod 'katello/candlepin',             '>= 5.0.0 < 5.1.0'
 
-mod 'katello/foreman_proxy_content', '>= 6.0.0 < 6.1.0'
-mod 'katello/katello',               '>= 7.0.0 < 7.1.0'
-
-# Katello devel specific modules
-mod 'katello/katello_devel',         :git => 'https://github.com/Katello/puppet-katello_devel'
+mod 'katello/foreman_proxy_content', '>= 7.0.0 < 7.1.0'
```

Comment:
This chnage will pull in a variety of changes:

 1) http://projects.theforeman.org/issues/21464 -- removal of pulp-oauth support, this requires multiple changes
 2) http://projects.theforeman.org/issues/21430 -- this would need to be re-aligned
 3) http://projects.theforeman.org/issues/22338 -- which is only available as of Pulp 2.15 I believe

Thus, I think we are going to have to do a 6.1.z release with the puppet-qpid support for dispatch router